swagger/src/main/java/grpcbridge/swagger/ParametersBuilder.java

```diff
@@ -132,21 +137,31 @@ public void onSimpleField(FieldDescriptor field, SimpleFieldType type) {
             // Ignore definition of repeated field.
             return;
         }
-        String name = fullPathName(field);
-        Location location = locator.getLocation(name);
+        Location location = locator.getLocation(fullPathName(field, true));
         if (location == Location.BODY) {
             return;
+        } else if (location == Location.PATH) {
+            pathParameters.add(field);
         }
-        parameters.add(Parameter.forSimpleField(name, location, field, config.isRequired(field)));
+        parameters.add(Parameter.forSimpleField(
+                fullPathName(field),
+                location,
+                field,
+                config.isRequired(field)));
     }
 
-    private String fullPathName(FieldDescriptor field) {
+    private String fullPathName(FieldDescriptor field, boolean isForLocation) {
```

Comment:
Without knowing the context here it is a bit difficult to understand why the `isForLocation` param is present. Would it make sense to name it something like `formatFieldName` and then create two additional helpers, e.g:
```
jsonPathTo(field) {
    return fullPathName(field, true)
}
protoPathTo(field) {
    return fullPathName(field, false)
}
```
and use the helpers above instead?

Reply:
I agree, will change it.